ocamlformat-help.txt

```diff
@@ -47,6 +47,12 @@ OPTIONS (CODE FORMATTING STYLE)
            assignment expression does not fit on a single line. The default
            value is end-line.
 
+       --break-before-in={always|auto}
+           Whether the line should break before the in keyword of a let
+           binding. always will always break the line before the in keyword.
+           auto will only break the line if the in keyword does not fit on
+           the previous line. The default value is always.
```

Comment:
`always` is only when it cannot fit on a single line otherwise there is no break. What about `fit-or-vertical` instead ?

Reply:
Fixed.